docs/01_cva6_user/ip-xact/cva6_csr.md

```diff
@@ -13,15 +13,15 @@ The fields within the ``fcsr`` can also be accessed individually through differe
 | 0 | NX | Inexact | read-write  | The accrued exception flags indicate the exception conditions that have arisen on any floating\-point arithmetic instruction since the field was last reset by software\. The base RISC\-V ISA does not support generating a trap on the setting of a floating\-point exception flag\.|
 
 ## Floating-Point Dynamic Rounding Mode Register 
```

Comment:
Do we explain elsewhere that the FP registers to not exist in the case where the top-level configuration/parameterization does not explicitly include support for floating point?